code/datums/components/wall_mounted.dm

```diff
@@ -41,7 +41,8 @@
  */
 /datum/component/wall_mounted/proc/on_examine(datum/source, mob/user, list/examine_list)
 	SIGNAL_HANDLER
-	examine_list += span_notice("\The [hanging_wall_turf] is currently supporting [span_bold("[parent]")]. Deconstruction or excessive damage would cause it to [span_bold("fall to the ground")].")
+	if (parent in view(user.client?.view || world.view, user))
```

Comment:
doesn't view() do this implicitly? or does it default to world and such.

Reply:
> doesn't view() do this implicitly? or does it default to world and such.

defaults to world.view I believe? Its not in the docs, I know that viewer() defaults specifically to world.view too, but stuff like sniper rifles exist so we wanna make sure

Reply:
I swear there's an explicit proc for this but I can't find it so eh

Reply:
> I swear there's an explicit proc for this but I can't find it so eh

wanna do viewers() then? (We'll be ignoring scopes and alike then)

Reply:
nawww